.azure/pipelines/ci.yml

```diff
@@ -664,6 +664,12 @@ stages:
         isTestingJob: true
         buildArgs: --all --test "/p:RunTemplateTests=false /p:SkipHelixReadyTests=true" $(_InternalRuntimeDownloadArgs)
         beforeBuild:
+        - bash: "brew uninstall openssl@1.0.2t"
```

Comment:
this really should strive to be conditional, or ignored;  it will fail once openssl@1.0.2t is not present.